docs/app/Examples/modules/Popup/Variations/PopupWidthExample.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Icon, Popup } from 'stardust'
+
+export default function () {
+  return (
+    <div>
+      <Popup
+        trigger={<Icon circular name='heart' />}
+        content='Hello. This is a wide pop-up which allows for lots of content
```

Comment:
No worries.  Our goal is to provide two APIs 1) a "Shorthand Props API" for the majority simplified use cases and 2) a sub component API as an escape hatch when you need access and control over the full markup.

It is worth noting, the two are mutually exclusive.  So, you cannot mix a shorthand `header` prop while also having `children` defined.  More [here](https://github.com/TechnologyAdvice/stardust/blob/master/.github/CONTRIBUTING.md#component-part-props) in the CONTRIBUTING.md.  Also, I plan to add more info to the docs on this, #561.

Here are some equivalent examples:

```jsx
<Popup content='The content' />
<Popup>
  The content
<Popup>
```
```jsx
<Popup header='My Header' content='The content' />
<Popup>
  <Popup.Header>My header</Popup.Header>
  <Popup.Content>The content</Popup.Content>
<Popup>
```

Most components have shorthand capabilities you can reference for more examples.